redhat-cop/casl-ansible
# Add deletion of Cinder Registry

#### What does this PR do?
This change will delete the auto-provisioned cinder registry volume when the `delete-cluster` playbook is called

#### How should this be manually tested?
1) run `end-to-end.yml` to provision the cluster on OpenStack with an inventory to auto-provision a registry volume and verify a {{ full_dns_domain }}-registry volume is provisioned in Openstack
2) run `delete-cluster.yml` and verify the volume is deleted upon completion

#### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #200 

#### Who would you like to review this?
cc: @redhat-cop/casl
